Description: Define _GNU_SOURCE to enable BSD and GNU features
Forwarded: not-needed
Author: Matej Vela <vela@debian.org>
Last-Update: 2011-05-13

Index: csh-20110313/Makefile
===================================================================
--- csh-20110313.orig/Makefile
+++ csh-20110313/Makefile
@@ -7,6 +7,7 @@
 
 PROG=	csh
 DFLAGS=-DBUILTIN -DFILEC -DNLS -DSHORT_STRINGS
+DFLAGS+=-D_GNU_SOURCE
 #CFLAGS+=-g
 #CFLAGS+=-Wall
 CFLAGS+=-I${.CURDIR} -I. ${DFLAGS}
